packages/api-server/src/watch.ts

```diff
@@ -30,8 +30,12 @@ build({ incremental: true }).then((buildResult) => {
     process.exit(0)
   })
 
+  const watchPaths = []
+  watchPaths.push(rwjsPaths.api.base)
+  watchPaths.push(rwjsPaths.redwoodConfig)
+
   chokidar
-    .watch(rwjsPaths.api.base, {
+    .watch(watchPaths, {
```

Comment:
Are you trying to restart the api server on a redwood.toml change?

And then your question is where to also to something similar on the dev server?

Reply:
Yeah, I did with the api server but need to see for the webpack dev server. I will try what David provided resources.